serverless.yml

```diff
@@ -0,0 +1,71 @@
+service: taco-gallery
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+  region: us-east-2
+  stage: dev
+  environment:
+    TESTNUM: 27
+    TESTSTR: hello
+    AUTH0_DOMAIN: pfernandom.auth0.com
+    AUTH0_CLIENTID: L7YiTnpuzmfa1W9spj7fIpwXgHKzSmNX
+
+# resources:
+#   Resources:
+#     TacosDB:
+#       Type: 'AWS::DynamoDB::Table'
+#       Properties:
+#         AttributeDefinitions:
+#           -
+#             AttributeName: id
+#             AttributeType: S
+#           -
+#             AttributeName: name
+#             AttributeType: S
+#         KeySchema:
+#           -
+#             AttributeName: id
+#             KeyType: HASH
+#           -
+#             AttributeName: name
+#             KeyType: RANGE
+#         ProvisionedThroughput:
+#           ReadCapacityUnits: 1
+#           WriteCapacityUnits: 1
+#         TableName: TacoGallery
+
+functions:
+  authorizerFunc:
+      handler: handler.auth
+  helloPublic:
+    handler: index.sayHello
+    events:
+      - http:
+          path: hello
+          method: post
+          cors: true
+  handleMessage:
+      handler: index.handleMessage
+      events:
+        - http:
+            path: handleMessage
+            method: post
+            integration: lambda
+            cors: true
+
+
+plugins:
+  # - serverless-dynamodb-local
+  - serverless-offline
+  # - serverless-webpack
+
+# custom:
```

Comment:
We can leave this commented, maybe be needed for the real project but not here

Reply:
we are using it soon to integrate with dynamoDB